OctopusDSC/OctopusDSCHelpers.ps1

```diff
@@ -1,7 +1,24 @@
 # Module contains shared code for OctopusDSC
 
-$octopusServerExePath = "$($env:ProgramFiles)\Octopus Deploy\Octopus\Octopus.Server.exe"
-$tentacleExePath = "$($env:ProgramFiles)\Octopus Deploy\Tentacle\Tentacle.exe"
+function Get-OctopusServerExePath {
+    $installLocation = (Get-ItemProperty -path "HKLM:\Software\Octopus\OctopusServer" -ErrorAction SilentlyContinue).InstallLocation
+
+    if ($installLocation -ne $null) {
+        return "$installLocation\Octopus.Server.exe"
+    }
+
+    return ""
```

Comment:
I wonder if we should fall back to the default paths here? Otherwise we'll hit weird errors, eg [here](https://github.com/OctopusDeploy/OctopusDSC/pull/281/files#diff-0154615783f6e2eb98f5b51cabb7a30c7924c9e930adea57a19427080f63e938R100-R103)